egs/iam/v1/run.sh

```diff
@@ -78,12 +78,12 @@ if [ $stage -le 4 ]; then
     data/lang exp/mono
 fi
 
-if [ $stage -le 5 ]; then
-  utils/mkgraph.sh --mono data/lang_test exp/mono exp/mono/graph
-
-  steps/decode.sh --nj $nj --cmd $cmd exp/mono/graph data/test \
-    exp/mono/decode_test
-fi
+#if [ $stage -le 5 ]; then
```

Comment:
are these commented-out parts supposed to be commented out?

Reply:
Yes, I intentionally commented out the decoding parts because the results from the first couple of stages (monophone/triphone) are not very good and I didn't think I needed to compute them. I can just pass their alignments on to the later CNN stages of the script.

Reply:
Rather than comment them out, you can set at the top
`decode_gmm=false`
and do things like
`if [ $stage -le 5 ] && $decode_gmm; then`